### PR TITLE
Adding ifxbgt60dev to documentation index for distro

### PR DESCRIPTION
**Pull request by Yang-IFX**
I'd like ifxbgt60dev to be indexed and documented on ros.org.

### CODE REVIEW

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -2483,6 +2483,22 @@ repositories:
       url: https://github.com/ethz-adrl/ifopt.git
       version: master
     status: maintained
+  ifxbgt60dev:
+    doc:
+      type: git
+      url: https://github.com/Yang-IFX/ifxbgt60dev.git
+      version: master
+    release:
```
Comment (tfoote) [inline noetic/distribution.yaml:2491]: You don't appear to have made a release. This whole stanza should not be included. bloom will generate it when you make a release following the first time release tutorial http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

**Review by tfoote — CHANGES_REQUESTED** [review]
Before this can be merged you'll also need to add proper documentation of you license choice. 

There must be a LICENSE file. Either each package in the repository should have a LICENSE file, or the source repository should have a top-level LICENSE file. If the repository is under multiple licenses, then it is encouraged (but not required) to have one LICENSE. file per license.
The license should be one of the open-source ones listed in https://opensource.org/licenses (preferably one of the "Popular licenses").
The license must be reflected in the package.xml file of all sub-packages in the repository.

The structure of your repository is a little unusual. Usually the package is either in the root if it's intended to be a single package repository, or all packages are in subdirectories of the root for easy discoverability. 

I would also strongly suggest adding at least a little bit of documenation to your package such as a description with real content in the package.xml and some content into your README. 
 For example I don't know what a ifx bgt60 is having read all the obviously discoverable documentation in the repository. Generating documentation and being indexed is much more useful if there's content to generate and index.

**Comment by Yang-IFX** [comment]
The necessary changes are done:

1. "There must be a LICENSE file. Either each package in the repository should have a LICENSE file, or the source repository should have a top-level LICENSE file. If the repository is under multiple licenses, then it is encouraged (but not required) to have one LICENSE. file per license.
The license should be one of the open-source ones listed in https://opensource.org/licenses (preferably one of the "Popular licenses").
The license must be reflected in the package.xml file of all sub-packages in the repository."

The missing license is added in the "package.xml" file.

2. "The structure of your repository is a little unusual. Usually the package is either in the root if it's intended to be a single package repository, or all packages are in subdirectories of the root for easy discoverability."

Since ifx_bgt60_device is the first package released now, a multi-package repository structure is chosen. 

3. "I would also strongly suggest adding at least a little bit of documenation to your package such as a description with real content in the package.xml and some content into your README." 

The basic information about ifxbgt60dev is added in README.md.

4. "You don't appear to have made a release. This whole stanza should not be included. bloom will generate it when you make a release following the first time release tutorial http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease"

The information for the release will be added later.

Thank you for your time and attention.

**Comment by tfoote** [comment]
Thanks for the update overall you've addressed all of it. One small thing for the license is that the first part of it says that you should literally have a file named `LICENSE` that contains the text of the license. So that people can know exactly what you're talking about.

**Pull request closed without merging**